mail_message_auto_expand/__openerp__.py

```diff
@@ -0,0 +1,13 @@
+﻿{
+    'name': 'Mail Message Auto Expand',
+    'version': '8.0',
+    'summary': 'Expands the Read More Automatically',
+    'description': '',
```

Comment:
Either a description, or the [README.rst template of OCA](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)